pkg/mountexecutor/mountexecutor.go

```diff
@@ -57,7 +57,7 @@ func NewMountExecutor(o *MountExecutorOptions) *MountExecutor {
 }
 
 // StartMounting starts the mounting
-func (m *MountExecutor) StartMounting(o *MountOptions) error {
+func (m *MountExecutor) StartMounting(o *MountOptions, valuesLogger klog.Logger) error {
```

Comment:
I think we should add the logger field in the `MountExecutor` struct, so that we don't change the method signature.

Reply:
I added it to MountOptions. I believe MountExecutor is created once and remains constant for all requests, whereas MountOptions are per request, like the valuesLogger.